src/amo/components/Link.js

```diff
@@ -18,6 +21,9 @@ export class LinkBase extends React.Component {
   }
 
   static defaultProps = {
+    // TODO: Make this default to `true` and force internal links to label
+    // themselves as not external?
+    external: false,
```

Comment:
Yeah, that's a good call.

Reply:
Just noticed the devhub link doesn't actually start with `http` because it doesn't need to (it's on the same domain).

For now I'll just keep the external flag if that's cool; we can improve this later but I don't think it's worth it right now.